quickstarts/aws/aws-elemental-mediatailor/config.yml

```diff
@@ -0,0 +1,34 @@
+slug: aws-elemental-mediatailor
+description: |-
+  ## What is AWS Elemental MediaTailor?
+  AWS Elemental MediaTailor is a channel assembly and personalized ad-insertion service for video providers to create linear over-the-top (OTT) channels using existing video content. The service then lets you monetize those channels or other live streams with personalized advertising.
+  ### Get started!
+  Start monitoring AWS Elemental MediaTailor by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-elemental-mediatailor-monitoring-integration/) to learn more about New Relic monitoring for Amazon MediaTailor.
+summary: |-
+  Monitor AWS Elemental MediaTailor by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: AWS Elemental MediaTailor
+documentation:
+  - name: AWS Elemental MediaTailor installation docs
+    description: |
+      Monitor AWS Elemental MediaTailor by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-elemental-mediatailor-monitoring-integration/
```

Comment:
Seems like this docs link is not live yet:
https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-elemental-mediatailor-monitoring-integration/

Lets hold until this is